manual/parts/vspace.tex

```diff
@@ -86,12 +86,9 @@ \subsubsection{x64}
 
 \subsubsection{AArch32}
 
-Like IA-32, Arm AArch32 realise the VSpace as a \texttt{PageDirectory}, which covers the entire
-4\,GiB address range, and a second-level \texttt{PageTable}. The second-level structures on AArch32
-cover 1\,MiB address ranges.
-
-Arm AArch32 processors have a two-level page-table structure.
-The top-level page directory covers a range of 4\,GiB and each page table covers a 1\,MiB range.
+Like IA-32, Arm AArch32 implements the VSpace object with a \texttt{PageDirectory} object which
+covers the entire 4\,GiB address range.  The second-level structures on AArch32 are
+\texttt{PageTable} objects and cover 1\,MiB address ranges.
```

Comment:
Technically on aarch32 hyp Page table objects cover 2MiB instead of 1MiB.

Reply:
You're right, even un-technically. I think this is just wrong and should be updated.

Reply:
Added the distinction between hyp and non-hyp.